[2022-08-19 23:08:38,361] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: sefa_dag.get_telegram_version scheduled__2020-01-01T03:25:00+00:00 [queued]>
[2022-08-19 23:08:38,378] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: sefa_dag.get_telegram_version scheduled__2020-01-01T03:25:00+00:00 [queued]>
[2022-08-19 23:08:38,379] {taskinstance.py:1376} INFO - 
--------------------------------------------------------------------------------
[2022-08-19 23:08:38,379] {taskinstance.py:1377} INFO - Starting attempt 1 of 1
[2022-08-19 23:08:38,381] {taskinstance.py:1378} INFO - 
--------------------------------------------------------------------------------
[2022-08-19 23:08:38,407] {taskinstance.py:1397} INFO - Executing <Task(PythonOperator): get_telegram_version> on 2020-01-01 03:25:00+00:00
[2022-08-19 23:08:38,424] {standard_task_runner.py:52} INFO - Started process 2630 to run task
[2022-08-19 23:08:38,437] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'sefa_dag', 'get_telegram_version', 'scheduled__2020-01-01T03:25:00+00:00', '--job-id', '430', '--raw', '--subdir', 'DAGS_FOLDER/sefa_dag.py', '--cfg-path', '/tmp/tmpevgwczos', '--error-file', '/tmp/tmpjn1vore3']
[2022-08-19 23:08:38,467] {standard_task_runner.py:80} INFO - Job 430: Subtask get_telegram_version
[2022-08-19 23:08:38,586] {task_command.py:371} INFO - Running <TaskInstance: sefa_dag.get_telegram_version scheduled__2020-01-01T03:25:00+00:00 [running]> on host fd7a784b9251
[2022-08-19 23:08:38,790] {taskinstance.py:1591} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=sefa_dag
AIRFLOW_CTX_TASK_ID=get_telegram_version
AIRFLOW_CTX_EXECUTION_DATE=2020-01-01T03:25:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2020-01-01T03:25:00+00:00
[2022-08-19 23:08:38,794] {python.py:173} INFO - Done. Returned value was: 1.24.0
[2022-08-19 23:08:38,827] {taskinstance.py:1420} INFO - Marking task as SUCCESS. dag_id=sefa_dag, task_id=get_telegram_version, execution_date=20200101T032500, start_date=20220819T230838, end_date=20220819T230838
[2022-08-19 23:08:38,901] {local_task_job.py:156} INFO - Task exited with return code 0
[2022-08-19 23:08:38,940] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
